program-analysis/echidna/basic/common-testing-approaches.md

```diff
@@ -180,15 +157,12 @@ contract SystemModel {
 }
 ```
 
-At this point, we should be able to compile our model without any dependency from the original codebase (everything necessary should be included in
-the model). Then, we can insert assertions to detect when the returned value exceeds a certain threshold.
+At this point, we should be able to compile our model without any dependency on the original codebase (everything necessary should be included in the model). We can then insert assertions to detect when the returned value exceeds a certain threshold.
 
-While developers or auditors can be tempted to quickly create tests using this technique there are certain disadvantages when creating models:
+While developers or auditors may be tempted to quickly create tests using this technique, there are certain disadvantages when creating models:
 
-- The code tested can be very different from the one we want to test: this can either introduce issues that are not real (false positives) or
-  hide real issues from the original code (false negatives). In the example, it is unclear if the state variable can take arbitrary values.
+- The tested code can be very different from what we want to test. This can either introduce unreal issues (false positives) or hide real issues from the original code (false negatives). In the example, it is unclear if the state variables can take arbitrary values.
 
-- The model will have a limited value if the code is changed since any modification in the original model will force a rebuild of the model,
-  and this should be manually performed.
+- The model will have limited value if the code changes since any modification to the original model will require rebuilding the model, a manual process.
```

Comment:
`a manual process` sounds incomplete here, maybe `require manually rebuilding the model` ?